components/p3a/brave_p3a_switches.h

```diff
@@ -17,7 +17,12 @@ constexpr char kP3ADoNotRandomizeUploadInterval[] =
     "p3a-do-not-randomize-upload-interval";
 
 // Interval between restarting the uploading process for all gathered values.
-constexpr char kP3ARotationIntervalSeconds[] = "p3a-rotation-interval-seconds";
```

Comment:
i don't think we should change this, some people (like QA) could got used to it and save in scripts or docs